### PR TITLE
[tests-only][full-ci] added cli test for changing password of non-existing user

### PR DESCRIPTION
**Pull request by S-Panta**
## Description
Adding test scenario for changing password of non existing user via cli

## Related Issue

- Part of https://github.com/owncloud/ocis/issues/9529

## How Has This Been Tested?

- test environment:
-  locally and CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/bootstrap/CliContext.php [hunk 1]
```diff
@@ -68,21 +68,24 @@ public function theAdministratorHasStartedTheServer(): void {
 	}
 
 	/**
-	 * @When the administrator resets the password of user :user to :password using the CLI
+	 * @When /^the administrator resets the password of (non-existing|existing) user "([^"]*)" to "([^"]*)" using the CLI$/
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/CliContext.php:71]: please, confirm if we use existent or existing

File: tests/acceptance/features/cliCommands/resetUserPassword.feature [hunk 2]
```diff
@@ -8,10 +8,17 @@ Feature: reset user password via CLI command
...
+    When the administrator resets the password of existing user "Alice" to "newpass" using the CLI
     Then the command should be successful
     And the command output should contain "Password for user 'uid=Alice,ou=users,o=libregraph-idm' updated."
     But the command output should not contain "Failed to update user password: entry does not exist"
     And the administrator has started the server
     And user "Alice" should be able to create folder "newFolder" using password "newpass"
     But user "Alice" should not be able to create folder "anotherFolder" using password "%alt1%"
+
+
+  Scenario: reset password of non-existing user
```
Comment (saw-jan) [inline tests/acceptance/features/cliCommands/resetUserPassword.feature:20]:
```suggestion
  Scenario: try to reset password of non-existing user
```

**Review by saw-jan — APPROVED** [review]

**Review by ScharfViktor — APPROVED** [review]

**Comment by ScharfViktor** [comment]
Resetting a password via cli is an admin feature. The most important case is to check that:
-  **admin** can change its own password. 
-  **admin** after renaming its name can change its own password 

I put this case to https://github.com/owncloud/ocis/issues/9529

**Comment by saw-jan** [comment]
>I put this case to https://github.com/owncloud/ocis/issues/9529

Thank you

**Pull request merged**